From 37432ef26b5793add709837450e68d2a16467908 Mon Sep 17 00:00:00 2001
From: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
Date: Fri, 23 Feb 2018 14:58:19 +0530
Subject: [PATCH 1468/2345] ASoC: Intel: Skylake: Remove unused host_cpr_params
 variable

In cnl_sdw_bra_pipe_cfg_cp(), host_cpr_params variable serves no useful
purpose hence remove its code reference.

Change-Id: I8efea650a5adf2ec5f91fb290b21d628c397e201
Acked-by: Vaibhav Agarwal <vaibhav.agarwal@intel.com>
Signed-off-by: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
---
 sound/soc/intel/skylake/skl-messages.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/sound/soc/intel/skylake/skl-messages.c b/sound/soc/intel/skylake/skl-messages.c
index 9b5c2b8..ed7f3a3 100644
--- a/sound/soc/intel/skylake/skl-messages.c
+++ b/sound/soc/intel/skylake/skl-messages.c
@@ -646,7 +646,7 @@ static int cnl_sdw_bra_pipe_cfg_cp(struct skl_sst *ctx,
 {
 	struct bra_conf *bra_data = &ctx->bra_pipe_data[mstr_num];
 	struct skl_pipe *link_cpr_pipe = NULL;
-	struct skl_pipe_params link_cpr_params, host_cpr_params;
+	struct skl_pipe_params link_cpr_params;
 	struct skl_module *host_cpr_mod = NULL, *link_cpr_mod = NULL;
 	struct skl_module_cfg *link_cpr_cfg = NULL, *host_cpr_cfg = NULL;
 	int ret;
@@ -827,8 +827,6 @@ static int cnl_sdw_bra_pipe_cfg_cp(struct skl_sst *ctx,
 
 	memcpy(host_cpr_cfg, link_cpr_cfg,
 			sizeof(struct skl_module_cfg));
-	memcpy(&host_cpr_params, &link_cpr_params,
-			sizeof(struct skl_pipe_params));
 
 	host_cpr_cfg->id.instance_id = 4;
 	host_cpr_cfg->id.pvt_id = skl_get_pvt_id(ctx,
@@ -840,7 +838,6 @@ static int cnl_sdw_bra_pipe_cfg_cp(struct skl_sst *ctx,
 	host_cpr_cfg->dev_type = SKL_DEVICE_HDAHOST;
 	host_cpr_cfg->hw_conn_type = SKL_CONN_SINK;
 	link_cpr_params.host_dma_id = (bra_data->cp_stream_tag - 1);
-	host_cpr_params.host_dma_id = (bra_data->cp_stream_tag - 1);
 	host_cpr_cfg->formats_config[SKL_PARAM_INIT].caps_size = 0;
 	host_cpr_cfg->m_in_pin = kcalloc(host_cpr_cfg->module->max_input_pins,
 					sizeof(*host_cpr_cfg->m_in_pin),
-- 
2.7.4

